Simplenote/SPAppDelegate.m

```diff
@@ -470,6 +475,9 @@ - (void)bucket:(SPBucket *)bucket didChangeObjectForKey:(NSString *)key forChang
                 if ([key isEqualToString:self.noteEditorViewController.note.simperiumKey]) {
                     [self.noteEditorViewController didDeleteCurrentNote];
                 }
+
+                [self alertListenerOfDeleteWithKey:key];
```

Comment:
Nipticky: Do you think we could drop the bridging APIs and invoke directly `publishController?`